src/tbp/monty/frameworks/environment_utils/transforms.py

```diff
@@ -590,14 +630,13 @@ def get_surface_from_depth(
         # sign), and apply it to the depth to get the semantic patch.
         th, flip_sign = self.get_on_surface_th(
             depth_patch,
-            min_depth_range=0.01,
-            default_on_surface_th=default_on_surface_th,
-            semantic_mask=semantic_mask,
+            semantic_patch,
+            0.01,
+            default_on_surface_th,
```

Comment:
Is there a reason you are not using key words here anymore? I prefer it looking like `min_depth_range=0.01,` instead of just `0.01,` since it is more readable and less error prone.